### ISSUE TITLE
Stacker's yaml linter rejects duplicated dict entries

### ISSUE THREAD

**Issue by Lowercases** [remind101/stacker#506]
A stack setup like
```
stacks:
  - name: stack1
    class_path: classpath1
    variables: &common_variables
      var1: value1
      var2: value2
  - name: stack2
    class_path: classpath2
    variables:
      <<: *common_variables
      var1: value1override
```
used to work, however a check for duplicated entries was introduced in https://github.com/remind101/stacker/pull/500 and "overriding" a previous value is no longer possible.

This may break existing setups.

**Comment by troyready** [comment]
This should be fixed in 1.1.3 (via #507 )

**Comment by phobologic** [comment]
Thanks @troyready
